src/main/java/com/to/ordermanagementservice/repository/impl/InMemoryOrderItemRepository.java

```diff
@@ -16,16 +16,16 @@ public class InMemoryOrderItemRepository implements OrderItemRepository {
 
     public InMemoryOrderItemRepository() {
         OrderItem orderItem11 = new OrderItem();
-        orderItem11.setId(11);
+        orderItem11.setId(1);
```

Comment:
please, set `id` here and in the rest of places to prev. values

Reply:
changed value